sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateEcKeyOptions.java

```diff
@@ -35,21 +35,35 @@ public CreateEcKeyOptions(String name) {
     /**
      * Get the curve.
      *
-     * @return The curve.
+     * @return The curve name.
      */
     public KeyCurveName getCurveName() {
         return this.curveName;
     }
 
     /**
-     * Set the curve.
+     * Set the curve name.
      *
-     * @param curveName The curve to set.
+     * @param curveName The curve name to set.
      *
      * @return The {@link CreateEcKeyOptions} object.
      */
     public CreateEcKeyOptions setCurveName(KeyCurveName curveName) {
         this.curveName = curveName;
+
+        return this;
+    }
+
+    /**
+     * Set the key size.
+     *
+     * @param keySize The key size to set.
+     *
+     * @return The updated {@link CreateEcKeyOptions} object.
+     */
+    public CreateEcKeyOptions setKeySize(Integer keySize) {
+        super.setKeySize(keySize);
```

Comment:
Is `keySize` exposed from the parent then? And is `setKeySize` public on the parent? It shouldn't be exposed, though if it is I realize it's too late to change now.